src/cloud/release-notes/mcd-release-notes.md

```diff
@@ -16,6 +16,19 @@ The release notes include:
 -  {:.new}New features
 -  {:.fix}Fixes and improvements
 
+## v1.2.1
+*Release date: December 21, 2020*<br/>
+
+-  {:.fix}Fixed PHP-FPM container health check by changing the `WEB_PORT` from `80` to `8080`.<!--MCLOUD-7232-->
+
+-  {:.fix}Fixed an error with invalid volume naming in developer mode.<!--MCLOUD-7442-->
+
+-  {:.new}Added options to change the value of NGINX `worker_processes` (option: `--nginx-worker-processes`) and NGINX `worker_connections` (option: `--nginx-worker-processes` ) for tls and web services. The parameter `worker_process` retains the ability to set the value `auto`.<!--MCLOUD-7259-->
```

Comment:
Should probably indicate these are options for the `build` command. Same for next line, too.